### PR TITLE
Fix Signs for water streams

### LINKED ISSUE #140: Signs_for_water_streams does not update water when sign is placed.
#130 Previously I got an error message when I ran the script. I fixed it after i opened an issue. I also updated it for the new 1.16 wood types. Now I'm getting a new error where the water is not updated. Screenshot attached. Updated 1.16 + error message script attached.

DISCLAIMER: I don't know scarpet. I just added the wood types at the start in quotation marks like the other wood types, and I just followed instructions others gave me.
[signs_for_water_streams.zip](https://github.com/gnembon/scarpet/files/5307500/signs_for_water_streams.zip)

### PR DESCRIPTION
**Pull request by Ghoulboy78**
Fixes #140

### CODE REVIEW

**Review by BisUmTo — APPROVED** [review]

File: programs/survival/signs_for_water_streams.sc [hunk 1]
```diff
@@ -1,65 +1,52 @@
...
-);
+// tested in Carpet 1.4.24
 
+// https://github.com/gnembon/fabric-carpet/releases
+
+// namespaces for items and placed signs differ, so map each wall_sign to it's item equivalent
+
+__config)(->{
+	'stay_loaded'->true
+};
```
Comment (altrisi) [inline programs/survival/signs_for_water_streams.sc:17]: I think you didn't test this when readding it...
```suggestion
__config() -> {
	'stay_loaded' -> true
};
```

**Review by altrisi — APPROVED** [review]

**Comment by Meeni-mc** [comment]
Mmm this also breaks compatibility with modded signs (e.g. Terrestria).

**Comment by altrisi** [comment]
Well, I guess you can add them manually to the map if they aren't many. Not sure why the heuristics were removed though.

**Comment by altrisi** [comment]
It seems like this has been based on the version previous to your PR, now that I'm looking at it. That may be why.

**Pull request merged**